src/masternode-payments.cpp

```
@@ -137,9 +137,16 @@ std::string GetRequiredPaymentsString(int nBlockHeight, const CDeterministicMNCP
         CTxDestination dest;
-        if (!ExtractDestination(payee->pdmnState->scriptPayout, dest))
-            assert(false);
-        strPayee = CBitcoinAddress(dest).ToString();
+        if (!ExtractDestination(payee->pdmnState->scriptPayout, dest)) {
+            std::string strScriptPayout = spark::ToStringSparkAddress(payee->pdmnState->scriptPayout);
+            if (!strScriptPayout.empty())
+                strPayee = strScriptPayout;
+            else
+                assert(false);
+        } else {
+            strPayee = CBitcoinAddress(dest).ToString();
```

cppcheck (2.10-2)

[performance] Variable 'blockHash' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)